flytekit/extras/sqlite3/task.py

```diff
@@ -84,16 +92,76 @@ def output_columns(self) -> typing.Optional[typing.List[str]]:
         c = self.python_interface.outputs["results"].column_names()
         return c if c else None
 
-    def execute(self, **kwargs) -> typing.Any:
+    def get_custom(self, settings: SerializationSettings) -> typing.Dict[str, typing.Any]:
+        return {
+            "query_template": self.query_template,
+            "inputs": "",
+            "outputs": "",
+            "uri": self.task_config.uri,
+            "compressed": self.task_config.compressed,
+        }
+
+    def get_command(self, settings: SerializationSettings) -> typing.List[str]:
+        return [
+            "pyflyte-manual-execute",
+            "--inputs",
+            "{{.input}}",
+            "--output-prefix",
+            "{{.outputPrefix}}",
+            "--raw-output-data-prefix",
+            "{{.rawOutputDataPrefix}}",
+            "--task_executor",
+            f"{SQLite3TaskExecutor.__module__}.{SQLite3TaskExecutor.__name__}",
+            "--task_template_path",
+            "{{.taskTemplatePath}}",
+        ]
+
+    def serialize_to_template(self, settings: SerializationSettings) -> task_models.TaskTemplate:
```

Comment:
why? This is confusing right? We have a base serialization method that does this already?